cmd/rancher-desktop-guestagent/main.go

```diff
@@ -181,7 +194,7 @@ func main() {
 	if *enableIptables {
 		group.Go(func() error {
 			// Forward ports
```

Comment:
The `Forward ports` comment isn't really useful. (This isn't blocking merge, it's just something silly I noticed.)

Reply:
> FWIW, it's relatively easy to build a enum flag thing:

Good call, I was always leaning towards changing the arg management to cobra but I went forward and did a quick cleanup for now.

Reply:
Actually, this approach will not work since iptables can be enabled in conjunction with docker or containerd under some conditions, e.g [older k8s ](https://github.com/rancher-sandbox/rancher-desktop/blob/02baf7c606c31f3ff5e460480e1259184a6157cc/pkg/rancher-desktop/backend/wsl.ts#L838)

Reply:
The chart says:
> When the iptables enabled in this case, the docker or containerd must be disabled.

That seems to be incorrect.